tests/utils/test_geometry_comparisons.py

```diff
@@ -104,6 +104,44 @@ def test_compare_global_poses_scaled_squares(self):
         pose_graphs_equal = geometry_comparisons.compare_global_poses(wTi_list, wTi_list_)
         self.assertTrue(pose_graphs_equal)
 
+    def test_compute_translation_to_direction_angle_is_zero(self):
+        i2Ui1_measured = Unit3(Point3(1, 0, 0))
+        wTi2_estimated = Pose3(Rot3(), Point3(0, 0, 0))
+        wTi1_estimated = Pose3(Rot3(), Point3(2, 0, 0))
+        self.assertEqual(
+            geometry_comparisons.compute_translation_to_direction_angle(i2Ui1_measured, wTi2_estimated, wTi1_estimated),
+            0.0,
+        )
+
+    def test_compute_translation_to_direction_angle_is_nonzero(self):
+        i2Ui1_measured = Unit3(Point3(0, 1, 0))
+        wRi2 = Rot3.RzRyRx(-np.deg2rad(90), 0, 0)  # x-axis points to -y in world frame
+        wRi1 = Rot3.RzRyRx(np.deg2rad(30), np.deg2rad(60), np.deg2rad(90))  # irrelevant
+        wTi2_estimated = Pose3(wRi2, Point3(0, 0, 0))
+        wTi1_estimated = Pose3(Rot3(), Point3(0, -1, 0))  # along -y axis
+        # estimated direction along x-axis and measured along y-axis in i2 frame.
+        self.assertTrue(
+            geometry_comparisons.compute_translation_to_direction_angle(i2Ui1_measured, wTi2_estimated, wTi1_estimated),
+            90.0,
```

Comment:
should it be 180 degrees, not 90 degrees @akshay-krishnan ? Since i2Ui1_measured is (0,1,0)

but poses suggest (0,-1,0)

Reply:
`i2Ui1_measured` is in i2 frame, and since this frame is rotated by -90 wrt world frame, (0, 1, 0) is actually (1, 0, 0) in the world frame. The angle between (0, -1, 0) and (1, 0, 0) in world frame is 90 degrees.

Reply:
I think roll, pitch, yaw makes it more confusing. I always end up wondering which one corresponds to rotation around X, Y and Z. But I do agree the RzRyRx API is odd. 
wRi1 is irrelevant, and it seems like too much effort for something irrelevant. I can make that an identity rotation and have the rx change for wRi2. 
I don't think I understand your suggestion on yaw, not roll. Let me make this change, and could we please speak in terms of x, y, z? :)